components/LearnCourse/LearnCourse.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <main class="learn-course">
+  <div class="learn-course">
```

Comment:
Could you double check this change? I think this is more an [article](https://html.spec.whatwg.org/#the-article-element) than a [div](https://html.spec.whatwg.org/#the-div-element)

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3441/commits/8813034834a0b3ef57c06bdf5a74df881100161f